checkov/kubernetes/graph_manager.py

```diff
@@ -23,11 +24,10 @@ def build_graph_from_source_directory(
         render_variables: bool = True,
         parsing_errors: dict[str, Exception] | None = None,
         download_external_modules: bool = False,
-        excluded_paths: list[str] | None = None,
-        graph_flags: K8sGraphFlags | None = None
+        excluded_paths: list[str] | None = None
     ) -> tuple[KubernetesLocalGraph, dict[str, list[dict[str, Any]]]]:
         definitions, definitions_raw = get_folder_definitions(source_dir, excluded_paths)
-        local_graph = self.build_graph_from_definitions(definitions=definitions, render_variables=False, graph_flags=graph_flags)
+        local_graph = self.build_graph_from_definitions(definitions=definitions, render_variables=False, graph_flags=self.graph_flags)
```

Comment:
if the graph_flags are saved as a property in the class why do you need to pass it to the `build_graph_from_definitions` function? you can use it there directly

Reply:
It's not necessary, I can remove it